pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform.CoreCLR/files/Microsoft.NETCore.UniversalWindowsPlatform.CoreCLR.props

```diff
@@ -0,0 +1,5 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <UseAppLocalCoreCLR>true</UseAppLocalCoreCLR>
```

Comment:
do we need an issue to remove this when AppX FW package comes through?

CC @rahku